admin/taxonomy/class-taxonomy-fields-presenter.php

```diff
@@ -186,32 +187,44 @@ private function get_help( $field_html, $help_text, $field_name ) {
 	 * @return string
 	 */
 	private function parse_help( $field_name, $help_text ) {
-		static $image_src;
+		return sprintf(
+			'<p id="%1$s" class="yoast-help-panel">%2$s</p>',
+			esc_attr( $field_name . '-help' ),
+			$help_text
+		);
+	}
 
-		if ( $image_src === null ) {
-			$image_src = plugins_url( 'images/question-mark.png', WPSEO_FILE );
+	/**
+	 * Returna the help toggle button
+	 *
+	 * @param string $button_text The help button text.
+	 * @param string $field_name  The target field.
+	 *
+	 * @return string
+	 */
+	private function get_help_button( $button_text, $field_name ) {
+		if ( $button_text !== '' ) {
+			return ' <button type="button" class="yoast_help yoast-help-button dashicons" id="' . esc_attr( $field_name . '-help-toggle' ) .
+				'" aria-expanded="false" aria-controls="' . esc_attr( $field_name . '-help' ) . '"><span class="screen-reader-text">' .
+				$button_text . '</span></button>';
 		}
 
-		return sprintf(
-			'<img src="%1$s" class="alignright yoast_help" id="%2$s" alt="%3$s" />',
-			$image_src,
-			esc_attr( $field_name . 'help' ),
-			esc_attr( $help_text )
-		);
+		return '';
 	}
 
 	/**
 	 * Returns the HTML for the row which contains label, help and the field.
 	 *
-	 * @param string $label The html for the label if there was a label set.
-	 * @param string $help  The html for the help, when it's there.
-	 * @param string $field The html for the field.
+	 * @param string $label       The html for the label if there was a label set.
+	 * @param string $help_button The text for the help button.
```

Comment:
This should be `The html for the help button`, because to get a help button you have to pass the HTML for it to this function.